Visitor/Arithmetic.php

```diff
@@ -285,7 +285,7 @@ public function getFunctions()
      * Get a function.
      *
      * @param   string  $name    Function name.
-     * @return  \Hoa\Consistency\Xcallable
+     * @return  \Hoa\Core\Consistency\Xcallable
```

Comment:
Nop, please, remove this. `Hoa\Core` does no longer exist (see http://blog.hoa-project.net/posts/36-the-nucleus-releases-serie.html).